elb_log_ingestor/main.py

```diff
@@ -19,16 +19,18 @@ def start_server():
     parser_stats = stats.ParserStats()
     shipper_stats = stats.ShipperStats()
 
+    shipper_count = int(os.environ.get("ELB_INGESTOR_SHIPPER_COUNT", "5"))
+
     elasticsearch_hosts = os.environ["ELB_INGESTOR_ELASTICSEARCH_HOSTS"]
     elasticsearch_hosts = elasticsearch_hosts.split(",")
     es_client = elasticsearch.Elasticsearch(
         elasticsearch_hosts, sniff_on_start=True, sniffer_timeout=60
     )
     server_address = get_server_address()
 
-    logs_to_be_processed = queue.Queue()
+    logs_to_be_processed = queue.Queue(maxsize=10)
```

Comment:
`logs_to_be_processed` is really log _files_ to be processed. Small is good here.